includes/WebRequest.php

```diff
@@ -1134,6 +1134,23 @@ public function isValidWriteRequest( \User $user ) {
 	public function isWikiaInternalRequest() {
 		return $this->getHeader( self::WIKIA_INTERNAL_REQUEST_HEADER ) !== false;
 	}
+
```

Comment:
Its inconsistent then; the PLATFORM change just above this one (isWikiaInternalRequest) isn't marked with "Wikia Change" for example.  Is there some recommendation for this?  Looking through recent commits in non-wikia directories under `includes` I see several changes that don't include any Wikia markings.  Of those that do, I see many formats:

* Start and end pairs of comments
* A single note in the function comment header
* Inline comments at the end of a line
* Many wordings "Wikia start", "Wikia change begin", "Wikia:", A Jira ticket number etc.
* Some begin/ends that include a JIRA ticket but the update is for a different ticket number